packages/backend/src/models/Channel.ts

```diff
@@ -93,4 +93,9 @@ export class MiChannel {
 		default: false,
 	})
 	public isSensitive: boolean;
+
+	@Column('boolean', {
+		default: true,
+	})
+	public canRenote: boolean;
```

Comment:
チャンネル内Renoteはできるから
```suggestion
	public allowRenoteToExternal: boolean;
```
の方が良さそう

Reply:
たしかに。関連箇所は全部allow～にリネームします

Reply:
以下で対応しました
59d3d49